pallet/src/lib.rs

```diff
@@ -158,11 +155,43 @@ pub mod pallet {
             <Authorities<T>>::put(authorities);
         }
 
-        pub fn next_session_authorities() -> Result<Vec<T::AuthorityId>, AlephApiError> {
+        pub fn next_session_authorities() -> Vec<T::AuthorityId> {
             Session::<T>::queued_keys()
                 .iter()
-                .map(|(_, key)| key.get(T::AuthorityId::ID).ok_or(AlephApiError::DecodeKey))
-                .collect::<Result<Vec<T::AuthorityId>, AlephApiError>>()
+                .map(|(_, key)| {
+                    key.get(T::AuthorityId::ID)
+                        .expect("Keys in storage decode correctly.")
+                })
+                .collect()
+        }
+
+        pub fn future_session_validator_ids(
+            session_id: u32,
```

Comment:
So this function either returns the next validators or the current validator set (ignoring the error for now). The naming is a little bit confusing I would assume that it always returns the next validator set, not sure what is an appropriate name here. `validator_ids_for_session`?

Reply:
I would say that it's pretty clear due to the argument, so imo the name is fine.